calm/db/migrate/20211030195131_create_announcements.rb

```diff
@@ -1,17 +1,19 @@
-class CreateMovies < ActiveRecord::Migration
+class CreateAnnouncements < ActiveRecord::Migration
   def up
-    create_table :movies do |t|
+    create_table :announcements do |t|
       t.string :title
+      t.string :author
       t.string :rating
```

Comment:
we don't need rating in announcements.

Reply:
We could change the name 'rating' to 'tag' or something. I didn't remove it because it could be used in the future.